apps/dialtone-documentation/docs/about/whats-new/posts/2024-4-11.md

```diff
@@ -0,0 +1,282 @@
+dist/lib/contact-row.cjs                           3.35 kB │ gzip: 1.28 kB │ map:  6.65 kB
+dist/lib/badge.cjs                                 3.55 kB │ gzip: 1.24 kB │ map:  7.58 kB
+dist/chunks/index-sdfB7Aok.js                      3.96 kB │ gzip: 1.76 kB │ map: 15.69 kB
+dist/chunks/keyboard_list_navigation-N74Bpdq7.js   3.99 kB │ gzip: 1.46 kB │ map: 16.58 kB
+dist/lib/select-menu.cjs                           4.08 kB │ gzip: 1.53 kB │ map: 11.75 kB
+dist/lib/callbar-button-with-popover.cjs           4.22 kB │ gzip: 1.61 kB │ map: 11.04 kB
+dist/lib/button.cjs                                4.27 kB │ gzip: 1.71 kB │ map: 13.29 kB
+dist/lib/feed-item-row.cjs                         4.36 kB │ gzip: 1.63 kB │ map: 12.51 kB
+dist/lib/ivr-node.cjs                              4.48 kB │ gzip: 1.83 kB │ map:  9.43 kB
+dist/lib/collapsible.cjs                           4.52 kB │ gzip: 1.70 kB │ map: 12.32 kB
+dist/lib/utils.cjs                                 4.56 kB │ gzip: 2.19 kB │ map: 17.15 kB
+dist/chunks/tab-7hJQSLFx.js                        5.36 kB │ gzip: 1.86 kB │ map: 13.03 kB
+dist/chunks/dropdown-zhMEz3bn.js                   5.53 kB │ gzip: 2.07 kB │ map: 16.79 kB
+dist/lib/modal.cjs                                 5.66 kB │ gzip: 2.10 kB │ map: 15.31 kB
+dist/lib/tooltip.cjs                               5.74 kB │ gzip: 2.25 kB │ map: 19.16 kB
+dist/lib/avatar.cjs                                5.87 kB │ gzip: 2.20 kB │ map: 19.60 kB
+dist/lib/attachment-carousel.cjs                   5.90 kB │ gzip: 1.84 kB │ map: 13.76 kB
+dist/lib/combobox-with-popover.cjs                 6.21 kB │ gzip: 2.35 kB │ map: 15.21 kB
+dist/chunks/index-2jPosQBn.js                      6.81 kB │ gzip: 2.26 kB │ map: 15.96 kB
+dist/lib/skeleton.cjs                              7.24 kB │ gzip: 2.10 kB │ map: 23.48 kB
+dist/lib/general-row.cjs                           8.41 kB │ gzip: 2.87 kB │ map: 18.36 kB
+dist/lib/input.cjs                                 8.62 kB │ gzip: 2.62 kB │ map: 22.12 kB
+dist/lib/combobox-multi-select.cjs                 9.04 kB │ gzip: 3.07 kB │ map: 22.41 kB
+dist/lib/message-input.cjs                        11.55 kB │ gzip: 3.59 kB │ map: 22.70 kB
+dist/lib/datepicker.cjs                           13.13 kB │ gzip: 3.47 kB │ map: 34.18 kB
+dist/lib/rich-text-editor.cjs                     14.56 kB │ gzip: 5.17 kB │ map: 52.99 kB
+dist/dialtone-vue.cjs                             14.98 kB │ gzip: 3.94 kB │ map:  0.10 kB
+dist/lib/editor.cjs                               14.98 kB │ gzip: 4.15 kB │ map: 28.14 kB
+dist/lib/popover.cjs                              14.98 kB │ gzip: 4.58 kB │ map: 41.03 kB
+dist/lib/emoji-picker.cjs                         23.10 kB │ gzip: 6.21 kB │ map: 50.83 kB
+✓ built in 2.15s
+```
+
+## Migration to dialtone >=9.27.1
+
+Replace any instances of the old imports with the corresponding new imports.
+Refer to the examples table below for clarity.
```

Comment:
We should maybe explain when you should use these imports (webpack vs vite vs vite with module federation)

Reply:
I'll try to add a short explanation as adding the full explanation here would be too much.

Reply:
yeah I'm thinking just like a simple

`webpack - import { DtEmoji } from '@dialpad/dialtone/vue3/emoji'`
`vite - import { DtEmoji } from '@dialpad/dialtone/vue3'`